<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="header" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
 background: #0E1426;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="cartIcon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
  background: gray;
}</string>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Users/User/Downloads/6c586905d23154b033d0df11175efce1 (1).png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="headingContainer" stretch="0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="heading">
           <property name="font">
            <font>
             <family>Ink Draft</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
  color: white;
}</string>
           </property>
           <property name="text">
            <string>Shopping Cart</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="headingBadgeContainer" stretch="1,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="headingBadge">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
 color: orange;
}</string>
             </property>
             <property name="text">
              <string>PRO</string>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="headingBadgeSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="userContainer" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="userName">
            <property name="text">
             <string>Guest</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="userIcon">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mainContainer" stretch="1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="sidebar" native="true">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
  background: #0E1426;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="cartButton">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
  color: white;
}</string>
           </property>
           <property name="text">
            <string> Cart</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/assets/clear.png</normaloff>:/assets/clear.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="historyButton">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
  color: white;
}</string>
           </property>
           <property name="text">
            <string> History</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>364</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="settingsButton">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
  color: white;
}</string>
           </property>
           <property name="text">
            <string> Settings</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="windows">
        <property name="styleSheet">
         <string notr="true">QWidget {
 background: cyan;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>571</width>
             <height>481</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1000000</width>
             <height>1000000</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget {
  background: #151A2C;
}</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="cartPage">
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="cartItems" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="cartItemsHeader" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>50</height>
                   </size>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QWidget" name="cartItemsSearchBox" native="true">
                     <property name="styleSheet">
                      <string notr="true">.QWidget {
  border: 1px solid rgba(255, 255, 255, 0.2);
  border-radius: 12px;
  margin: 5px;
}</string>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,1">
                      <property name="spacing">
                       <number>3</number>
                      </property>
                      <property name="leftMargin">
                       <number>5</number>
                      </property>
                      <property name="topMargin">
                       <number>5</number>
                      </property>
                      <property name="rightMargin">
                       <number>5</number>
                      </property>
                      <property name="bottomMargin">
                       <number>5</number>
                      </property>
                      <item>
                       <spacer name="cartItemsSpacer">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeType">
                         <enum>QSizePolicy::Minimum</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>2</width>
                          <height>10</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QLabel" name="cartItemsSearchIcon">
                        <property name="minimumSize">
                         <size>
                          <width>24</width>
                          <height>24</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>29</width>
                          <height>29</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">QLabel {
  padding: 2px;
}</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="pixmap">
                         <pixmap resource="resources.qrc">:/assets/search.png</pixmap>
                        </property>
                        <property name="scaledContents">
                         <bool>true</bool>
                        </property>
                        <property name="margin">
                         <number>3</number>
                        </property>
                        <property name="indent">
                         <number>0</number>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="cartItemsSearchLine">
                        <property name="styleSheet">
                         <string notr="true">QLineEdit {
  color: white;
}</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="frame">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="cartItemsSearchClear">
                        <property name="maximumSize">
                         <size>
                          <width>30</width>
                          <height>30</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true"/>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset resource="resources.qrc">
                          <normaloff>:/assets/clear.png</normaloff>:/assets/clear.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>18</width>
                          <height>18</height>
                         </size>
                        </property>
                        <property name="autoDefault">
                         <bool>false</bool>
                        </property>
                        <property name="flat">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="cartItemsAddButton">
                     <property name="minimumSize">
                      <size>
                       <width>105</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>100</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton {
  background: #38BDF8;
  color: black;
  padding: 10px 20px;
  border-radius: 12px;
}</string>
                     </property>
                     <property name="text">
                      <string>Add Item</string>
                     </property>
                     <property name="flat">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="cartItemsDisplay" native="true">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="cartSummary" native="true">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="cartSummaryHeader" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>50</height>
                   </size>
                  </property>
                  <widget class="QLabel" name="label">
                   <property name="geometry">
                    <rect>
                     <x>10</x>
                     <y>9</y>
                     <width>101</width>
                     <height>31</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>Shopping List</string>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="cartSummaryList"/>
                </item>
                <item>
                 <widget class="QWidget" name="cartSummarySummary" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>100</height>
                   </size>
                  </property>
                  <widget class="QLabel" name="cartSummaryTotalLabel">
                   <property name="geometry">
                    <rect>
                     <x>10</x>
                     <y>10</y>
                     <width>63</width>
                     <height>20</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>Total:</string>
                   </property>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="historyPage"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
